python/lvmdrp/core/image.py

```diff
@@ -2497,6 +2444,8 @@ def glueImages(images, positions):
         error=full_CCD_error,
         mask=full_CCD_mask,
         header=images[0].getHeader(),
+        individual_frames=images[0].getIndividualFrames(),
+        slitmap=images[0].getSlitmap(),
```

Comment:
There are a bunch of calls to `getSlitmap()` but that function only returns the `self._slitmap` property.  Where are you reading in and setting the fibermap?  I don't see any calls to `read_fibermap` or imports of `fibermap`.

Reply:
I think you already took care of that in the `run_drp` script, here: https://github.com/sdss/lvmdrp/blob/intslitmap/python/lvmdrp/functions/run_drp.py#L306

I'm relying on that to be able to get something useful when I call `getSlitmap`.

Reply:
Can you point me to the line of code where `self._slitmap` gets set?  I couldn't find it.

Reply:
There is a method `setSlitmap` here: https://github.com/sdss/lvmdrp/blob/intslitmap/python/lvmdrp/core/image.py#L2344